### PR TITLE
Fix types

### PR DESCRIPTION
**Pull request by procot**

### CODE REVIEW

**Comment by procot** [comment]
@vitalets @stamoern Please check this PR.

File: types/react-subscription.d.ts [hunk 1]
```diff
@@ -1,9 +1,9 @@
 
 import Subscription = require('./subscription');
```
Comment (vitalets) [inline types/react-subscription.d.ts]: Should not we switch `require` to `import` here as well?

**Review by vitalets — CHANGES_REQUESTED** [review]

File: types/react-subscription.d.ts (hunk 1 above)
Comment (procot) [reply types/react-subscription.d.ts]: Possible. For consistency. Whatever you say)

Comment (vitalets) [reply types/react-subscription.d.ts]: let's do it!

**Review by vitalets — APPROVED** [review]

**Pull request merged**